test/cpp/ext/otel/otel_plugin_test.cc

```diff
@@ -849,19 +851,22 @@ TEST_F(OpenTelemetryPluginOptionEnd2EndTest, ServerOnlyPluginOption) {
   const auto& attributes =
       data["grpc.client.attempt.duration"][0].attributes.GetAttributes();
   EXPECT_EQ(attributes.size(), 3);
-  EXPECT_EQ(attributes.find("key"), attributes.end());
+  EXPECT_THAT(
+      attributes,
+      ::testing::Not(::testing::Contains(std::make_pair("key", "value"))));
   // Verify server side metric
   ASSERT_EQ(data["grpc.server.call.duration"].size(), 1);
   const auto& server_attributes =
       data["grpc.server.call.duration"][0].attributes.GetAttributes();
   EXPECT_EQ(server_attributes.size(), 3);
   EXPECT_EQ(absl::get<std::string>(server_attributes.at("key")), "value");
-}
+}  // namespace
```

Comment:
This is the end of a test, not the end of a namespace.

Reply:
Thanks!